iojs.org/aliases.json

```diff
@@ -129,5 +129,8 @@
   ] },
     { "from": "cna-discussion-list", "to": [
       "michael_dawson@ca.ibm.com"
-  ] }
+  ] },
+    { "from": "security.ecosystem@nodejs.org", "to": [
+      "nodesecurity-f6dff0c98b32758c@forwarding.hackerone.com"
```

Comment:
Since the team got renamed, the address changed. Needs to be updated to `nodejs-ecosystem-f6dff0c98b32758c@forwarding.hackerone.com`.

Reply:
Done, thanks!